docs/prms/responsenet.rst

```diff
@@ -0,0 +1,11 @@
+ResponseNet
+===========
+
+ResponseNet is a pathway reconstruction algorithm which finds subnetworks using integer linear programming,
+considering an interactome as a way to send 'flow' from sources to targets.
+See the `original paper <https://doi.org/10.1038/ng.337>`_ and the corresponding reimplementation of it:
+https://github.com/Reed-CompBio/ResponseNet.
+
+ResponseNet takes one optional parameter:
+
+* gamma: (int) the size of the output graph: more gamma means more 'flow' gets passed along starting from the sources.
```

Comment:
I would say this "controls" the size of the output, because otherwise it could be misinterpreted as the output size in units of nodes or edges.